Docs: Add dashboard dto information (#114459)
grafana/grafana: docs/sources/developer-resources/api-reference/http-api/dashboard.md

- **403** – Access denied
- **404** – Not Found

### Retrieving additional access information

`GET /apis/dashboard.grafana.app/v1beta1/namespaces/:namespace/dashboards/:uid/dto`

> Retrieves a dashboard with additional access information, including if it is a public dashboard and the request users permissions on the dashboard. The `GET` response will include an additional `access` section with this data.

Reviewer: ```suggestion
Retrieves a dashboard with additional access information.

The `GET` response includes an additional `access` section with data such as if it's a public dashboard, or the request user permissions. 
```

Reviewer: @stephaniehingtgen I'm not sure of what the last bit is supposed to mean - the user permissions for the requests?

Author: it returns the permissions the user who sent the http request has on that particular dashboard (such as admin or editor permissions)

`GET /apis/dashboard.grafana.app/v1beta1/namespaces/:namespace/dashboards/:uid/dto`

Retrieves a dashboard with additional access information.

> The `GET` response includes an additional `access` section with data such as if it's a public dashboard, or the request user permissions. 

Reviewer: ```suggestion
The `GET` response includes an additional `access` section with data such as if it's a public dashboard, or the dashboard permissions (admin, editor) of the user who made the request. 
```
